setup.py

```diff
@@ -266,7 +269,7 @@ def generate_level(self, prefix, parser, Archiver):
             if self.generate_level(command + " ", parser, Archiver):
                 continue
 
-            with open('docs/usage/%s.rst.inc' % command.replace(" ", "_"), 'w') as doc:
+            with open('docs/usage/%s.rst%s' % (command.replace(" ", "_"), ".inc" if command != "borgfs" else ""), 'w') as doc:
```

Comment:
is that needed? guess we could also have usage/borgfs.rst including the .inc it would generate without that change.

Reply:
ok, will try that